README.md

```diff
@@ -90,12 +90,18 @@ WP_STASH_DRIVER=\Stash\Driver\FileSystem
 WP_STASH_DRIVER_ARGS='{"path":"\/var\/www\/cache","dirSplit":1}'
 ```
 
-Caching to a memcached server at `localhost`:
+Caching with a memcached server using the `memcached` host name (e.g. in DDEV):
 ```
 WP_STASH_DRIVER=\Stash\Driver\Memcache
 WP_STASH_DRIVER_ARGS='{"servers":["memcached","11211"]}'
 ```
 
+Caching with a Redis server using the `memcached` host name (e.g. in DDEV):
```

Comment:
Shouldn't this be `redis` instead?
```suggestion
Caching with a Redis server using the `redis` host name (e.g. in DDEV):
```

Reply:
You are absolutely right. Copy/paste mistake :see_no_evil: